website/src/views/login.js

```diff
@@ -22,20 +21,37 @@ export default function Login() {
         );
     }, []);
 
+    // Sanitize the input by escaping HTML special characters
+    function sanitizeInput(input) {
+        if (!input) return input; // Handle null or undefined inputs
+        return input.replace(/[&<>"']/g, function(match) {
+        // Replace special HTML characters with their respective HTML entities
+        switch (match) {
+            case '&': return '&amp;';
+            case '<': return '&lt;';
+            case '>': return '&gt;';
+            case '"': return '&quot;';
+            case "'": return '&#39;';
+            default: return match;
+        }
+        });
+    }
```

Comment:
Use a sanitization library -- never write your own sanitizer

Reply:
If we are using react, it should already be sanitizing the input for us.